gbpservice/nfp/orchestrator/modules/service_orchestrator.py

```diff
@@ -692,8 +624,8 @@ def create_network_function(self, context, network_function_info):
         }
         network_function = self.db_handler.create_network_function(
             self.db_session, network_function)
-        nfp_log.add_meta(self, meta_id=network_function['id'])
 
+        nfp_logging.store_logging_context(meta_id=network_function['id'])        
```

Comment:
Why we are setting meta_id only for create, delete functions, what about other functions
pt_add, pt_remove, ptg_add, ptg_remove functions ?